data/vectorojay.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Vector Ojay",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Photographer, transitioning to Tech || Software Developer(TypeScript, React). Versatile. Inquisitive. Passionate about Designs and UI. Eager for knowledge."
+  "avatar": "https://github.com/vectorojay.png"
```

Comment:
Avatars are not required as of now . It is directly picked up from GH actions. Linking a full example JSON file from the linkfree docs https://linkfree.eddiehub.io/docs/full-profile-example

Reply:
Also Add a comma `,` here as well at the end of line

Reply:
Thanks a lot, @stevenovak123